core-services/release-controller/_releases/release-okd-scos-4.17.json

```diff
@@ -36,7 +36,7 @@
       "upgrade": true,
       "upgradeFromRelease": {
         "candidate": {
-          "stream": "okd-scos",
+          "stream": "okd",
```

Comment:
so where is the `-scos` coming from ?

Reply:
The logic that assembles the stream name is located here:
https://github.com/openshift/release-controller/blob/c556d485536ef41361f681c950b2fc6fb8d3b16b/cmd/release-controller/sync_verify.go#L221
So, in this case, the `-scos` comes from the configuration here:
https://github.com/openshift/release/blob/4e3137cb95921c9fe3b00b29ad250829e99a22f5/core-services/release-controller/_releases/release-okd-scos-4.17.json#L3